test/test_cursor.py

```diff
@@ -1222,6 +1223,64 @@ def assertCursorKilled():
         else:
             self.assertEqual(0, len(listener.started_events))
 
+    @client_context.require_failCommand_appName
+    def test_timeout_kills_cursor_asynchronously(self):
+        # Kill any cursors possibly queued up by previous tests.
+        gc.collect()
+        self.client._process_periodic_tasks()
```

Comment:
Since we're using a new client for this test, these gc.collect/_process_periodic_tasks lines can be removed.

Reply:
`test_close_kills_cursor_synchronously` also uses a new client but still has the gc.collect/_process_periodic_tasks lines. Should that test also be updated in a different ticket?

Reply:
Yes good catch!